e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/AgentsE2eTest.java

```diff
@@ -97,6 +99,12 @@ void shouldReturnAgents()
                 .then().extract().response();
 
         // then
+
+        LoggedRequest loggedRequest = WireMock.findAll(WireMock.getRequestedFor(WireMock.anyUrl())).get(0);
+
+        assertNotNull(loggedRequest);
+        assertEquals("/agents?source=alfresco-dummy-source-id-0a63de491876", loggedRequest.getUrl());
```

Comment:
should we use the endpoint with "/integrations/agents?sourceId={sourceId}" instead "/agents?source={sourceId}" ?

Reply:
see that was mentioned above
LGTM